src/2.0/schema/schema.json

```diff
@@ -221,6 +221,21 @@
       },
       "additionalProperties": true
     },
+    "chainDetail": {
```

Comment:
Not necessary to change if you think otherwise, I'm just thinking if we should rename `chainDetail` to `network`?

Reply:
ok, i think type for blockchain is a bit confusing, so i also changed it to chain to determine which blockchain is being used.